CRM/Upgrade/Incremental/php/FiveSixtyOne.php

```diff
@@ -63,6 +63,16 @@ public function upgrade_5_61_alpha1($rev): void {
     $this->addTask(ts('Create index %1', [1 => 'civicrm_cache.UI_group_name_path']), 'addIndex', 'civicrm_cache', [['group_name', 'path']], 'UI');
     $this->addTask(ts('Create index %1', [1 => 'civicrm_cache.index_expired_date']), 'addIndex', 'civicrm_cache', [['expired_date']], 'index');
     $this->addTask(ts('Update Saved Mapping for contribution import', [1 => $rev]), 'convertMappingFieldsToApi4StyleNames', $rev);
+    $this->addTask(
+      'Add option group for file_type_id in file table',
+      'addOptionGroup',
+      [
+        'name' => 'file_type',
+        'title' => ts('File Type'),
+        'data_type' => 'Integer',
+        'is_reserved' => 1,
+      ]
```

Comment:
Test failure is:
```
Too few arguments to function CRM_Upgrade_Incremental_Base::addOptionGroup(  
  ), 2 passed in /home/jenkins/bknix-dfl/build/build-0/web/sites/all/modules/  
  civicrm/CRM/Queue/Task.php on line 101 and exactly 3 expected
```

Looks like you need to add another argument here which is an array of option values.